book/src/building/swift-packages/README.md

```diff
@@ -238,3 +233,38 @@ cd SwiftProject
 swift run
 # You should see "Hello from Rust!" in your terminal.
 ```
+
+# Usage on Linux
+Usage on Linux is a little bit different than on macOS. This guide will help you set up SwiftPM and Cargo. The general workflow is:
+* Set up Package.swift so that "cargo build" is called (You can use something like this here: "https://stackoverflow.com/questions/26971240/how-do-i-run-a-terminal-command-in-a-swift-script-e-g-xcodebuild" to invoke cargo during a swift build)
+* Set up swift-bridge according to the guide (see chapter "swiftc and cargo")
+* Link against swift-bridge and your library, like the one below:
+```swift
+targets: [
```

Comment:
Let's paste a complete file, unless there are so many lines that it's hard to understand.

This helps the reader better orient themselves towards what they're looking at.

Also, say what this file is. Explicitly say that this is a `Package.swift`

i.e. "like the `Package.swift` file below:"